### ISSUE TITLE
Lab1B

### ISSUE THREAD

**Issue by swptmp** [MicrosoftLearning/AZ-120-Planning-and-Administering-Microsoft-Azure-for-SAP-Workloads#9]
We had an issue with lab 1B where it appears some changes to the template is causing the DC deployment to fail.  Also, it seems the _artifacts Location url looks to be incomplete.  Below is what I added to our published labs to aid the student:

Please note: On step 7 and 8 below, the template you are using has changed slightly. You will need to serarch for "defaultValue": "Standard_DS2_v2" and replace it with "defaultValue": "Standard_D4s_v3,"

On step 9, the _artifacts Location: should be entered as https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/active-directory-new-domain-ha-2-dc/azuredeploy.json

Also on step 9, Please update Ad VM Size to be Standard_D4s_v3

**Comment by polichtm** [comment]
Did you manage to take the note of the error message generated during the failed deployment? If so, could you post it here?
rgds
Marcin

**Comment by swptmp** [comment]
The first thing they are claiming is confusing is they are looking for "adVMSize":, and it  actually reads "defaultValue". It's easy enough to figure out but we got complaints: 
![image](https://user-images.githubusercontent.com/37548845/98245192-daa55a80-1f70-11eb-9205-93ec24dbd4a2.png)

I just ran the lab two times without making the changes and now the VMs are deploying without error, so I'm going to go with Azure was being stupid.  Last week I was getting the error consistently.  So it looks like the only suggested change now would be to rewrite the part about adVMSize.
